### PR TITLE
Fix translate.escapeString to actually replace all occurrances

### PR DESCRIPTION
**Pull request by axic**
Actually fix #318.

### CODE REVIEW

File: test/fixtureAsmJson.output [hunk 1]
```diff
@@ -1,104 +1,104 @@
...
+      PUSH 0			ne her
+      ADDRESS 			\n\n\n 
+      PUSH FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF			\n\n\n  f
+      AND 			\n\n\n  f
+      PUSH 26121FF0			\n\n\n  f
+      SWAP1 			\n\n\n  function f
+      POP 			\n\n\n  function f
+      PUSH 100000000000000000000000000000000000000000000000000000000			\n\n\n  function f
+      MUL 			\n\n\n  function f
+      SWAP1 			ionally\n\n\n  function f
```
Comment (chriseth) [inline test/fixtureAsmJson.output]: This looks wrong

File: test/fixtureAsmJson.sol [hunk 2]
```diff
@@ -1,4 +1,7 @@
 contract C {
+  // Leaving an empty line here intentionally
+
+
   function f() returns (bytes4) {
```
Comment (chriseth) [inline test/fixtureAsmJson.sol]: no public?

Comment (axic) [reply test/fixtureAsmJson.sol]: This is not compiled, it is just for formatting.

**Comment by chriseth** [comment]
Could you change this so that the comment is moved further down? Then the test would be proper already for the first `PUSH1 0x40`.

File: test/fixtureAsmJson.output (hunk 1 above)
Comment (chriseth) [reply test/fixtureAsmJson.output]: It was correct before. It might be that `\n` is counted as two bytes?

File: test/fixtureAsmJson.sol (hunk 2 above)
Comment (chriseth) [reply test/fixtureAsmJson.sol]: I don't understand, there is assembly above. I thought it would be the test expectation.

Comment (axic) [reply test/fixtureAsmJson.sol]: It is not compiled by the compiler, only used by the formatting tool.

**Comment by chriseth** [comment]
I currently cannot verify that the test actually checks the change.

**Comment by axic** [comment]
The reason this is wrong that forgot to update `fixtureAsmJson.json`.

**Pull request merged**